doc/manual/installation/uninstalling-nix.xml

```diff
@@ -0,0 +1,56 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      version="5.0"
+      xml:id="ch-uninstalling-nix">
+
+<title>Uninstalling Nix</title>
+
+<para>The following steps describes how to remove Nix from your system.</para>
+
+<para>First of all you need to remove the <filename>/nix</filename>
+directory:</para>
+
+<screen>sudo rm -rf /nix</screen>
+
+<para>You also need to remove some files from your home directory:</para>
+
+<screen>
+rm -rf $HOME/.nix-profile
+rm -rf $HOME/.nix-defexpr
+rm -rf $HOME/.nix-channels
+rm -rf $HOME/.config/nixpkgs
+</screen>
+
+<para>Then you need to remove the line that references Nix from your
+<filename><envar>$HOME</envar>/.profile</filename>. Assuming that your username
+is <literal>user</literal> it should look something like this:</para>
+
+<screen>
+if [ -e /home/user/.nix-profile/etc/profile.d/nix.sh ]; then . /home/user/.nix-profile/etc/profile.d/nix.sh; fi # added by Nix installer
+</screen>
+
+<para>There might also be some configuration files in
+<filename>/etc/nix</filename> that you need to delete:</para>
+
+<screen>
+sudo rm -rf /etc/nix
+</screen>
+
+<para>Finally, on a multi-user installation of Nix, you will also need to remove
+the <literal>nixbld[0-9]</literal> users.</para>
+
+<screen>
+sudo deluser nixbld0
```

Comment:
This is system-specific. On NixOS (and I think most Linux distributions) the command is `userdel`. Also it can be more elegantly expressed using a for loop, e.g. `for $((i = 0; i < 10; i++)); do deluser nixbld$i; done`.

Reply:
Good catch, I'll update the PR.

Reply:
Although, strictly speaking, that for loop is also system-specific, isn't it? It doesn't work in my shell (fish).